ci/check-documentation.py

```diff
@@ -114,6 +116,22 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
+        # check <omz_dir> reference validity
+
+        for code_span in doc_page.code_spans():
+            if code_span.startswith(omz_reference_prefix):
+                target_path_rel = code_span[len(omz_reference_prefix):]
+                target_path = OMZ_ROOT / target_path_rel
+
+                if ".." in Path(target_path_rel).parts:
```

Comment:
You shouldn't convert `target_path_rel` to `Path` _here_. This variable represents a path, so it should contain a `Path` object already. You should move the `Path` call to the variable initialization.